### ISSUE TITLE
Add man pages and/or help text for commands

### ISSUE THREAD

**Issue by openjck** [openjck/git-fzf#1]
This article explains how this can be accomplished:

https://www.atlassian.com/git/articles/extending-git

**Comment by openjck** [comment]
Once the Python implementation is done, I might split this out into two tickets: one for help text (`--help`) and one for man pages. I think `argparse` makes the former very easy.

**Comment by openjck** [comment]
Split into #14 and #15.
